package.json

```diff
@@ -1,32 +1,65 @@
 {
-  "name": "mento-sdk",
-  "version": "1.0.0",
-  "description": "The SDK for Mento Protocol",
-  "main": "index.js",
+  "name": "@mentoprotocol/sdk",
+  "version": "0.1.0",
+  "license": "MIT",
+  "author": "Mento Labs",
+  "main": "dist/index.js",
+  "module": "dist/sdk.esm.js",
+  "typings": "dist/index.d.ts",
+  "files": [
+    "dist",
+    "src"
+  ],
   "scripts": {
-    "lint": "yarn prettier",
-    "lint:check": "yarn prettier:check && yarn lint:check",
-    "postinstall": "husky install",
-    "prettier": "prettier --config \"./.prettierrc.yml\" --write \"**/*.{json,md,ts,js,yml}\"",
-    "prettier:check": "prettier --config \"./.prettierrc.yml\" --check \"**/*.{json,md,ts,js,yml}\""
+    "analyze": "size-limit --why",
+    "build": "dts build",
+    "lint": "dts lint",
+    "prepare": "dts build",
+    "size": "size-limit",
+    "start": "dts watch",
+    "test": "dts test",
+    "coverage": "dts test --coverage"
   },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/mento-protocol/mento-sdk.git"
+  "husky": {
+    "hooks": {
+      "pre-commit": "dts lint"
+    }
   },
-  "author": "Mento Labs",
-  "license": "ISC",
-  "bugs": {
-    "url": "https://github.com/mento-protocol/mento-sdk/issues"
+  "prettier": {
```

Comment:
Nit, consider moving this to a prettier config file, as is the more common convention

Reply:
@bayological I believe the repo already had this and you moved it here when setting up dts. Any particular reason or should we just move it to a config file?

Reply:
No, this is just how it came with the template. I'm for moving to a separate file.